bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/internal/module/handler/GroupCommandTriggerHandler.java

```diff
@@ -143,10 +130,4 @@ public void dispose() {
         super.dispose();
         eventSubscriberRegistration.unregister();
     }
-
-    @Override
-    public boolean apply(Event event) {
-        logger.trace("->FILTER: {}", event.getTopic());
-        return event.getTopic().startsWith("openhab/items/");
-    }
 }
```

Comment:
Technically yes. In practice, no - we're listening for ItemAddedEvent, ItemRemovedEvent, and ItemCommandEvent. Those events are _always_ sent to openhab/items/<item>/{added|removed|command} (because they can only be created via ItemEventFactory, and those are the topics that are used by the factory). And the receive() method in this module ignores the topic anyway, going by type, and then for ItemCommandEvents it gets the item, and ensures it's a member of the group specified. In other words, the configuration for the module is still maintained. Event if some rogue component sent a look-alike ItemCommandEvent to openhab/somethingthatsnotitems/bogus, it would get to the receive() method, which would then ignore it because the `instanceof` would detect that it's not actually an ItemCommandEvent.